Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -95,7 +95,6 @@
 #include "DolphinQt/GameList/GameList.h"
 #include "DolphinQt/Host.h"
 #include "DolphinQt/HotkeyScheduler.h"
-#include "DolphinQt/InfinityBase/InfinityBaseWindow.h"
```

Comment:
Looks like you have rebased and removed the new Infinity Base window on accident